nschecker/notify.go

```diff
@@ -20,18 +18,19 @@ type Notifier interface {
 
 // SlackNotifier struct is construct an slack message.
```

Comment:
I didn't notice it before but "is construct" is not valid English, i guess.

"SlackNotifier handles notification to slack" or something is better.